packages/core/videoClient.ts

```diff
@@ -23,9 +23,17 @@ const getVideoAdapters = async (withCredentials: CredentialPayload[]): Promise<V
 
   for (const cred of withCredentials) {
     const appName = cred.type.split("_").join(""); // Transform `zoom_video` to `zoomvideo`;
-    const app = await appStore[appName as keyof typeof appStore]();
+    const appImportFn = appStore[appName as keyof typeof appStore];
 
-    if (app && "lib" in app && "VideoApiAdapter" in app.lib) {
+    // Static Link Video Apps don't exist in packages/app-store/index.ts(it's manually maintained at the moment) and they aren't needed there anyway.
+    const app = appImportFn ? await appImportFn() : null;
+
+    if (!app) {
```

Comment:
This was the reason the booking was failing. So, it would gracefully handle this flow now.